### PR TITLE
Update analytics tutorial

### PR DESCRIPTION
**Pull request by slavipande**
Automatic commit: Move 'btp-app-analytics-setup-use' from QA to Production

### CODE REVIEW

File: tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md [hunk 1]
```diff
@@ -0,0 +1,527 @@
...
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Overview)]
+SAP Fiori elements is a framework that comprises the most commonly used floor plans and is designed to:
+
+1. Speed up development by reducing the amount of front-end code needed to build SAP Fiori apps.
+2. Drive UX consistency and compliance with the latest SAP Fiori design guidelines.
+
+SAP Fiori elements for OData version 2 (V2) offers **Analytical list page (ALP)** as a separate floor plan for a unique way to analyze data step by step from different perspectives, to investigate a root cause through drill-down, and to act on transactional content. On the other hand, in SAP Fiori elements for OData V4, the **Analytical List Page (ALP)** is not a separate floor plan, but rather a "flavor" of the **List Report Object Page** floor plan. Hence, we can configure a **List Report Object Page** floor plan in such a way that it can serve as an "ALP flavor" report page.
```
Comment (MichaelCzcz) [inline tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md]: Please don't use "we" unless you are referring to SAP

File: tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md [hunk 2]
```diff
@@ -0,0 +1,527 @@
...
+tags: [ tutorial>beginner, software-product-function>sap-cloud-application-programming-model, programming-tool>node-js, software-product>sap-business-technology-platform, software-product>sap-fiori-tools, software-product>sapui5]
+primary_tag: software-product-function>sap-cloud-application-programming-model
+---
+
+
+
+## Prerequisites
+ - [Prepare Your Development Environment for CAP](btp-app-prepare-dev-environment-cap)
+ - Before you start with this tutorial, you have two options:
+    - Follow the instructions in **Step 16: Start from an example branch** of [Prepare Your Development Environment for CAP](btp-app-prepare-dev-environment-cap) to checkout the [`cap-roles`](https://github.com/SAP-samples/cloud-cap-risk-management/tree/cap-roles) branch.
```
Comment (MichaelCzcz) [inline tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md]: We want people to complete our tutorials. So, I would not tell them right away that they can skip a whole group of tutorials, and simply add this as a side note to your second option. Or you could put it as a note below the prerequisites section.

File: tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+
+    > In case you get an error launching the Application Generator, see the [SAP Fiori tools FAQ](https://help.sap.com/viewer/42532dbd1ebb434a80506113970f96e9/Latest/en-US) on SAP Help Portal to find a solution.
+
+2. Choose application type **SAP Fiori elements** and floor plan **List Report Object Page**.
+
+       ![V4 Template](vscv4template_fea.png)
+
+    > Why not use the Analytical List Page floor plan?
+
+    > Although we are using SAP Fiori elements for OData V4, you will notice that there is still a floor plan **Analytical List Page** in the previous screen. However, it is due to be removed from the Fiori Generator UI, so we are not going to use it for this tutorial.
```
Comment (MichaelCzcz) [inline tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md]: Please don't use "we" unless you are referring to SAP

File: tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+
+3. You should be able to see the generated resultant visualization.
+
+    !![Dashboard](fea_result_page.png)
+
+[DONE]
+[ACCORDION-END]
+---
+[ACCORDION-BEGIN [Step 9: ](Add your app to the launch page)]
+Let us add the `Risk Analysis` application to the launch page where we already have the `Risks` and `Mitigations` apps.
```
Comment (MichaelCzcz) [inline tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md]: Please don't use "we" unless you are referring to SAP

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thank you for this new tutorial. Please address the requested changes.

File: tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md (hunk 2 above)
Comment (slavipande) [reply tutorials/btp-app-analytics-setup-use/btp-app-analytics-setup-use.md]: Hi @MichaelCzcz,

I left another comment below, too. I'll be opening another PR and will transfer your comment there.

Best regards,
Svetoslav

**Comment by slavipande** [comment]
> Thank you for this new tutorial. Please address the requested changes.

Hi @MichaelCzcz,

Not a new tutorial, just a small update on a folder name mentioned in an already published CAP tutorial - [Create an SAP Fiori Elements-Based Analytical UI for your CAP Application](https://developers.sap.com/tutorials/btp-app-analytics-setup-use.html)

I guess my fork was not updated and this is why the diff shows it as something new. I'll update my fork accordingly and open another PR with the updates.

I'll also take care to address your remarks there.

Best regards,
Svetoslav

**Pull request closed without merging**